evaluate-job.sh

```diff
@@ -10,14 +10,14 @@ set -x
 
 # on_error () {
  	# on failure ship everything to s3
-# 	aws s3 sync . s3://almond-research/${owner}/models/${project}/${experiment}/${model}/failed_eval/
+# 	aws s3 sync . s3://${s3_bucket}/models/${project}/${experiment}/${model}/failed_eval/
```

Comment:
This is commented out so it doesn't matter, but you would still want the owner here, right?